tests/conftest.py

```diff
@@ -91,6 +91,61 @@ def get_all_classes(
     return set(all_classes)
 
 
+@pytest.fixture()
+def narwhalified_transformers_dict():
```

Comment:
Just raising this as we are setting precedents for a big piece of work so want to evaluate options.

Another way to do this would be to set cls.narwhalified as an attribute in the test class for a method.  I think this dict is better as it has the added bonus of showing us a list all in one place of which ones have been converted.

advantage of cls attribute would be that we wouldn't need to keep importing the fixture.  I guess it would also mean each method could be handled separately so could potentially refactor transform but leave fit for later (not sure if this would actually work).  Cant see this being terribly useful even if it did work though.

Reply:
to be fair, I could be open to the cls attribute as then info on whether/not narwhalified is less hidden from users... but yes would lose that nice collated view of which had been updated/have more places to keep track of updating. Happy for you to have final call, what do you think?

Reply:
cool think that is better, actioned here https://github.com/lvgig/tubular/pull/328/commits/e3114d99e18b8615a28d30edb3e107e20de385b2

Reply:
updated the attr name here https://github.com/lvgig/tubular/pull/328/commits